localflavor/id_/id_choices.py

```diff
@@ -1,42 +1,43 @@
 from django.utils.translation import ugettext_lazy as _
 
-#: Indonesia does not have an official Province code standard.
-#: I decided to use unambiguous and consistent (some are common) 3-letter codes.
-#: http://id.wikipedia.org/wiki/Daftar_provinsi_Indonesia
+# Provincial code in Indonesia doesn't comply with the standard 3-letter province code
+# data taken from the official page of Indonesian internal affair ministry
+# http://www.kemendagri.go.id/pages/data-wilayah
```

Comment:
I get a 404 page not found error when I try to load this URL. Has this page been removed since you made this pull request?